### PR TITLE
add `rexagod`

### PR DESCRIPTION
**Pull request by rexagod**
Signed-off-by: Pranshu Srivastava <rexagod@gmail.com>

### CODE REVIEW

**Comment by dgrisonnet** [comment]
@rexagod could you update the lists to match exactly the current list of leads?

**Comment by rexagod** [comment]
@dgrisonnet ACK :)

File: SECURITY_CONTACTS [hunk 1]
```diff
@@ -14,3 +14,4 @@ brancz
 logicalhan
 ehashman
 dashpole
+rexagod
```
Comment (dgrisonnet) [inline SECURITY_CONTACTS:16]: could you also update the list of security contacts to match the current approvers?

**Review by dgrisonnet — APPROVED** [review]
/lgtm
/approve

**Pull request merged**